docs/user/user/pings/submitting-custom-pings.md

```diff
@@ -0,0 +1,67 @@
+# Submitting a custom ping
```

Comment:
Sorry for being annoying, but this should be in the API reference section. (I am fine with a follow-up).